Rom.py

```diff
@@ -1588,6 +1588,27 @@ def hint_text(dest, ped_hint=False):
             hint = dest.pedestal_hint_text if dest.pedestal_hint_text else "unknown item"
         else:
             hint = dest.hint_text if dest.hint_text else "something"
+
+        # Corrections to the location of the chest in cross doorshuffle.
+        if world.doorShuffle[player] in ['crossed'] and not ped_hint:
+            if dest.name in ['Eastern Palace - Boss', 'Desert Palace - Boss', 'Tower of Hera - Boss', 'Skull Woods - Boss', 'Turtle Rock - Boss',
```

Comment:
Maybe if " - Boss" in dest.name" would be fine.

Reply:
Thank you.